crates/js-sys/src/lib.rs

```diff
@@ -4987,6 +4987,21 @@ macro_rules! arrays {
                 self.raw_copy_to(dst);
             }
 
+            /// Copy the contents of the source Rust slice into this
+            /// JS typed array.
+            ///
+            /// This function will efficiently copy the memory from within
+            /// the wasm module's own linear memory to this typed array.
+            ///
+            /// # Panics
+            ///
+            /// This function will panic if this typed array's length is
+            /// different than the length of the provided `src` array.
+            pub fn copy_from(&mut self, src: &[$ty]) {
+                assert_eq!(self.length() as usize, src.len());
+                unsafe { self.set(&$name::view(src), 0) }
```

Comment:
I suppose this needs a comment explaining with this `unsafe` block is actually safe? Something like:
```rust
// This is safe because the `set` function copies from its TypedArray argument
```
Suggestions are welcome! 😁